Neos.ContentGraph.DoctrineDbalAdapter/src/DoctrineDbalContentGraphProjection.php

```diff
@@ -195,6 +198,8 @@ public function apply(EventInterface $event, EventEnvelope $eventEnvelope): void
             RootNodeAggregateWithNodeWasCreated::class => $this->whenRootNodeAggregateWithNodeWasCreated($event, $eventEnvelope),
             SubtreeWasTagged::class => $this->whenSubtreeWasTagged($event),
             SubtreeWasUntagged::class => $this->whenSubtreeWasUntagged($event),
+            WorkspaceWasDiscarded::class => null, // only needed for GraphProjectorCatchUpHookForCacheFlushing
+            WorkspaceWasPartiallyDiscarded::class => null, // only needed for GraphProjectorCatchUpHookForCacheFlushing
```

Comment:
Maybe we could clarify the comments slightly along the lines of:
```
// the following two events are not actually handled, but we need to include them in {@see canHandle()} in order to trigger the catch up hooks for those (i.e. {@see GraphProjectorCatchUpHookForCacheFlushing}). This can be removed with https://github.com/neos/neos-development-collection/issues/4992